tests/PregTest.php

```diff
@@ -158,7 +159,7 @@ public function testSplitFailing()
             'PhpCsFixer\\PregException'
         );
         $this->expectExceptionMessage(
-            'Error occurred when calling preg_split.'
+            'Invalid PCRE pattern ""'
```

Comment:
can we have an example with sth else than `""` as well ?

Reply:
@keradus This is the original fixture IIRC. I'll take a look to put this more specifically under test w/ some more fixtures maybe on Preg::match specifically, I assume you're looking for more examples here which I think is a valid point.

Reply:
I am looking for example that would present the value of changes you are doing - now, we know it's there, but without explicit utest of it, it can be broken accidentally in future.
I understand you had to adjust existing tests to pass changed code.
It would be also great to add new cases to cover newly added functionality (even if the functionality is "only" better error messages)